Abc.MoqComplete/Abc.MoqComplete/CompletionProvider/SuggestMockProvider.cs

```diff
@@ -50,7 +53,16 @@ protected override bool AddLookupItems(CSharpCodeCompletionContext context, IIte
         private static ILookupItem GetLookupItem(CSharpCodeCompletionContext context, string proposedCompletion)
         {
             var lookupItem = CSharpLookupItemFactory.Instance.CreateKeywordLookupItem(context, proposedCompletion, TailType.None, PsiSymbolsThemedIcons.Variable.Id);
-            lookupItem.WithInitializedRanges(context.CompletionRanges, context.BasicContext);
+
+            var node = context.NodeInFile;
+            while (!(node is ICSharpArgument) && node != null) node = node?.Parent;
+            if (node != null)
+            {
+                var arg = (ICSharpArgument)node;
+                var range = arg.GetExtendedDocumentRange();
+                lookupItem.SetRanges(context.CompletionRanges.WithReplaceRange(range));
+            }
```

Comment:
Should we add something like `else { lookupItem.WithInitializedRanges(context.CompletionRanges, context.BasicContext); }` here?

Reply:
Why not. The initialized ranges doesn't work for some reason here and always return an empty range, but it wouldn't hurt to do that.